tools/respecDocWriter.js

```diff
@@ -250,9 +263,6 @@ function makeConsoleMsgHandler(page) {
           console.warn(colors.warn(`🚨 ${output}`));
           break;
       }
-      if (abortOnError || abortOnWarning) {
```

Comment:
We still need to exit the process with the right code based on either a warning or error.

Reply:
Ah sorry, I see you move it to the top. Ignore this comment.